### PR TITLE
refactor(unlock-app): Add validation to the name field on configurator

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -45,4 +45,4 @@
       "last 1 safari version"
     ]
   }
-}
```
Comment (julien51) [inline docs/package.json:48]: We need to fix this :D

**Review by julien51 — APPROVED** [review]
That's nice! Much easier than I thought!

File: unlock-app/src/components/interface/locks/CheckoutUrl/elements/LocksForm.tsx [hunk 2]
```diff
@@ -496,16 +548,7 @@ export const LocksForm = ({
                             <MetadataList />
                           ) : (
                             <div className="grid items-center grid-cols-1 gap-2 mt-2 rounded-xl">
-                              <DynamicForm
```
Comment (julien51) [inline unlock-app/src/components/interface/locks/CheckoutUrl/elements/LocksForm.tsx:499]: Do we still use that in other places? Would it be hard to move the validation in this one?

Comment (searchableguy) [reply unlock-app/src/components/interface/locks/CheckoutUrl/elements/LocksForm.tsx:499]: Yes, I need to think a bit more about refactoring this down to more manageable parts.

**Review by julien51 — APPROVED** [review]

**Pull request merged**